.nvmrc

```diff
@@ -0,0 +1 @@
+v8.11.2
```

Comment:
how is this related to adding a `.coafile` ?

Reply:
Related to the updated travis that support coala